domains/swami.json

```diff
@@ -6,6 +6,9 @@
       "email": "adityaedu22@gmail.com"
     },
   "record": {
-          "CNAME": "adityasworld.vercel.app"
+          "CNAME": "adityasworld.vercel.app",
+          "TXT": {
+            "_vercel": "vc-domain-verify=swami.is-a.dev,ebd6de23b5512663a728"
+          }
```

Comment:
1. you can't use CNAME with other records
2. This is not how you register a verification subdomain, you should create another fil `_vercel.swami.json` and put the verification TXT record there, see https://is-a.dev/docs/vercel

Reply:
my bad, I'll create new PR!!